docs/model-dev-guide/best-practices/_index.rst

```diff
@@ -1,75 +0,0 @@
-################
- Best Practices
-################
-
-***************************************
- General Tips for the Trial Definition
-***************************************
-
-Do:
-
--  Use framework abstractions to implement learning rate scheduling instead of directly changing the
-   learning rate. See `tf.keras.optimizers.schedules.LearningRateSchedule
-   <https://www.tensorflow.org/api_docs/python/tf/keras/optimizers/schedules/LearningRateSchedule>`__
-   and :class:`determined.pytorch.LRScheduler` as examples.
-
--  For code that needs to download artifacts (e.g., data, configurations, pretrained weights),
-   download to a `tempfile.TemporaryDirectory <https://docs.python.org/3/library/tempfile.html>`__
-   unique to the Python process. This will avoid race conditions when using distributed training, in
-   which Determined executes multiple Python processes in the same task container.
```

Comment:
this was always a horrible idea, and using a filelock can reduce the bandwidth of the download and resulting diskspace by a factor of up to `slots_per_trial`.  It's never been a best practice... even though there were those at the time who insisted that file locks were "too hard" for end users to understand.